examples/gnrc_networking/udp.c

```diff
@@ -97,16 +102,22 @@ static void send(char *addr_str, char *port_str, char *data, unsigned int num,
             ip = gnrc_pkt_prepend(ip, netif_hdr);
         }
         /* send packet */
-        if (!gnrc_netapi_dispatch_send(GNRC_NETTYPE_UDP, GNRC_NETREG_DEMUX_CTX_ALL, ip)) {
+        if (!gnrc_netapi_dispatch_send(GNRC_NETTYPE_UDP,
+                                       GNRC_NETREG_DEMUX_CTX_ALL, ip)) {
             puts("Error: unable to locate UDP thread");
             gnrc_pktbuf_release(ip);
             return;
         }
-        /* access to `payload` was implicitly given up with the send operation above
+        /* access to `payload` was implicitly given up with the send operation
+         * above
          * => use temporary variable for output */
         printf("Success: sent %u byte(s) to [%s]:%u\n", payload_size, addr_str,
                port);
+#if IS_USED(MODULE_ZTIMER_MSEC)
+        ztimer_sleep(ZTIMER_MSEC, delay);
+#else
         xtimer_usleep(delay);
```

Comment:
Why not use `xtimer_msleep()` and let `ztimer_xtimer_compat` handle the rest?

Reply:
Because I wanted to keep the changes to existing code to an absolute minimum.

Reply:
Well, this sounds like an argument for using `xtimer_msleep()`, doesn't it?

Reply:
No, since the existining code uses microsecond scale. See also [this discussion](https://github.com/RIOT-OS/RIOT/pull/16343#discussion_r614846476).

Reply:
Now this will sleep for 1000000 µs with xtimer but 1000000 **ms** with ztimer by default